packages/airnode-adapter/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const solidityBaseTypes = ['uint256', 'int256', 'bool', 'bytes32', 'address', 'bytes', 'string'] as const;
+export const artificialTypes = ['string32'] as const;
+export const baseResponseTypes = [...solidityBaseTypes, ...artificialTypes] as const;
+
+export const MULTIPLE_PARAMETERS_DELIMETER = ',';
+export const PATH_DELIMETER = ',';
```

Comment:
Should this be `,`? I couldn't find where it's used.

Reply:
> Should this be .? 

It should.

>  I couldn't find where it's used.

Nowhere - I forgot to use it... But I've fixed it in https://github.com/api3dao/airnode/pull/673/files#diff-4940026c48ea44c3e3afc561a628cc4a8b15cc39341356e9eba334e3c268dd29R6